app/dashboard/views.py

```diff
@@ -3074,6 +3080,23 @@ def authenticate_idena(request, handle):
         }
     })
 
+@login_required
+def recheck_brightid_status(request, handle):
+    is_logged_in_user = request.user.is_authenticated and request.user.username.lower() == handle.lower()
+    if not is_logged_in_user:
+        return JsonResponse({
+            'ok': False,
+            'msg': f'Request must be for the logged in user'
+        })
+    
+    profile = profile_helper(handle, True)
```

Comment:
Do we want to allow any user to be able to run this? or every user only runs this for his profile + admin | is_staff can run for other users??

Reply:
I think this should only be available for the logged in user, there is a job that kicks off every hour that performs this action to update bright_id_status to the db so not sure if we need the staff role to do this as well.